markdown_writer.py

```diff
@@ -160,7 +161,16 @@ def write_to_markdown(
             # Replace any whitespace
             issue.title = issue.title.strip()
 
-            file.write(f"| " f"{issue.title} | " f"{issue.html_url} |")
+            if non_mentioning_links:
+                file.write(
+                    f"| {issue.title} | "
+                    f"{issue.html_url.replace('https://github.com', 'https://www.github.com')} |"
+                )
+            else:
+                file.write(
+                    f"| {issue.title} | "
+                    f"{issue.html_url} |"
+                )
```

Comment:
```suggestion
                file.write(f"| {issue.title} | " f"{issue.html_url} |")
```

This is the fix to satisfy the linter.

Reply:
Thanks, sorry I missed out the `black` lint. Should be all fixed now!